packages/components/src/components/organisms/Table/index.tsx

```diff
@@ -1 +1,2 @@
+/* istanbul ignore file */
```

Comment:
Why do we need this? We should rather configure jest to exclude `index.tsx` files in my opinion

Reply:
Yeah - I was wondering if we want to remove all the `index.tsx` files from coverage or not...
`table` slices for example have all the logic on the `index.tsx`.

Shall we move it then and remove all the index.tsx from the coverage?

Reply:
Yeah I would suggest to move it and remove all index from coverage. We can do it in another PR 😄